### PR TITLE
Switch from nose2 to pytest

### PR DESCRIPTION
**Pull request by sven1103**
Addresses #32

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,10 +15,10 @@ install:
   - sudo ln -s /tmp/nextflow/nextflow /usr/local/bin/nextflow
   # Set up nf-core and test/coverage modules
   - cd ${TRAVIS_BUILD_DIR}
-  - sudo pip install .
-  - sudo pip install codecov nose nose2
+  - pip install .
+  - pip install codecov nose pytest pytest-cov
```
Comment (ewels) [inline .travis.yml]: Do we still need `nose`?

Comment (sven1103) [reply .travis.yml]: only for the `@raises` annotation so far, I can search for a replacement

Comment (sven1103) [reply .travis.yml]: Done

**Comment by ewels** [comment]
🎉

**Pull request merged**